src/fidesops/graph/config.py

```diff
@@ -394,6 +394,8 @@ class Collection(BaseModel):
     fields: List[Field]
     # an optional list of collections that this collection must run after
     after: Set[CollectionAddress] = set()
+    # An optional list of dependent fields that need to be queried together
+    grouped_inputs: Optional[List[str]] = []
```

Comment:
Should we make this a `Set[str]` so that the inputs are de-duped?

Reply:
sure, i can make it a set, but I'll still have it be input as a list, and transformed to a set before persisting here.